includes/validation/class-amp-validation-error-taxonomy.php

```diff
@@ -1647,28 +1650,34 @@ public static function filter_tag_row_actions( $actions, WP_Term $tag ) {
 				);
 			}
 
-			// @todo We should consider reversing the order.
-			// Only add the 'Reject' and 'Accept' links to the index page, not the individual URL page.
-			$sanitization = self::get_validation_error_sanitization( json_decode( $term->description, true ) );
-			if ( 'edit-tags.php' === $pagenow && self::VALIDATION_ERROR_ACK_REJECTED_STATUS !== $sanitization['term_status'] ) {
-				$actions[ self::VALIDATION_ERROR_REJECT_ACTION ] = sprintf(
-					'<a href="%s">%s</a>',
-					wp_nonce_url(
-						add_query_arg( array_merge( [ 'action' => self::VALIDATION_ERROR_REJECT_ACTION ], compact( 'term_id' ) ) ),
-						self::VALIDATION_ERROR_REJECT_ACTION
-					),
-					esc_html__( 'Reject', 'amp' )
-				);
-			}
-			if ( 'edit-tags.php' === $pagenow && self::VALIDATION_ERROR_ACK_ACCEPTED_STATUS !== $sanitization['term_status'] ) {
-				$actions[ self::VALIDATION_ERROR_ACCEPT_ACTION ] = sprintf(
-					'<a href="%s">%s</a>',
-					wp_nonce_url(
-						add_query_arg( array_merge( [ 'action' => self::VALIDATION_ERROR_ACCEPT_ACTION ], compact( 'term_id' ) ) ),
-						self::VALIDATION_ERROR_ACCEPT_ACTION
-					),
-					esc_html__( 'Accept', 'amp' )
-				);
+			// Only add the 'Remove' and 'Keep' links to the index page, not the individual URL page.
+			if ( 'edit-tags.php' === $pagenow ) {
+				$sanitization = self::get_validation_error_sanitization( json_decode( $term->description, true ) );
+
+				if ( self::VALIDATION_ERROR_ACK_ACCEPTED_STATUS !== $sanitization['status'] ) {
+					$actions[ self::VALIDATION_ERROR_ACCEPT_ACTION ] = sprintf(
+						'<a href="%s">%s</a>',
+						wp_nonce_url(
+							add_query_arg( array_merge( [ 'action' => self::VALIDATION_ERROR_ACCEPT_ACTION ], compact( 'term_id' ) ) ),
+							self::VALIDATION_ERROR_ACCEPT_ACTION
+						),
+						esc_html(
+							self::VALIDATION_ERROR_NEW_ACCEPTED_STATUS === $sanitization['term_status'] ? __( 'Confirm remove', 'amp' ) : __( 'Remove', 'amp' )
```

Comment:
Should this be `Confirm Removal`? Feels more 'English' to me. But not a native speaker.

Reply:
That is my gut sense as well. Nevertheless, I can't think of a similar word for “keep”, so that's why I left it as a bare verb. “Confirm keepal” doesn't work.

Reply:
Maybe “Confirm removed” and “Confirm kept” would be better.

Reply:
Yes, I think so, because “confirm removed” has more or a sense that an action has been performed, whereas “confirm removal” has more of a timeless sense. And using the past participle means we can use “confirm kept”, and be consistent with other places we use the past participles in the markup status dropdowns.

Reply:
Done in 803d00f0